tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -535,11 +537,14 @@ void test_parse_errors() {
           "an outflow-type boundary condition, you must use that."));
   using TDMO = domain::creators::sphere::TimeDependentMapOptions;
```

Comment:
The [sphere test failed](https://github.com/sxs-collaboration/spectre/actions/runs/6664974239/job/18113638093?pr=5576) for one of the builds still.